docs/src/main/asciidoc/cache.adoc

```diff
@@ -324,8 +324,8 @@ Here's what your cache configuration could look like:
 [source,properties]
 ----
 quarkus.cache.caffeine."foo".initial-capacity=10 <1>
-quarkus.cache.caffeine."foo".maximum-size=20 <1>
-quarkus.cache.caffeine."foo".expire-after-write <1>
```

Comment:
I think the callouts were on purpose. Do you have a warning/error? /cc @gwenneg

Reply:
But then they should be <1>, <2>, ... Because if not it looks really strange when they are rendered.

Reply:
There's a missing property value indeed, but I don't see any problem with the callouts. The same one is repeated three times on purpose since the same cache is being configured and it doesn't generate any warning/error AFAIK.

Reply:
![image](https://user-images.githubusercontent.com/10584698/73854371-bd512d80-4832-11ea-96f1-eefa79f0e657.png)

This looks fine to me (except for the missing value).

Reply:
Well now we are entering to the subjective things for me it is weird, but of course I understand that for somebody else this looks good :)

Reply:
I wouldn't mind removing the 2nd and 3rd callout if it makes the doc clearer :)

Reply:
In the end, it is my personal opinion, but of course, I might be wrong.